components/scripts/lib/build-scan-online.sh

```diff
@@ -48,7 +48,7 @@ fetch_single_build_scan() {
   local build_scan_url="$1"
 
   local build_scan_data
-  build_scan_data="$(fetch_build_scan_data brief_logging "${build_scan_url}")"
+  build_scan_data="$(fetch_build_scan_data verbose_logging "${build_scan_url}")"
```

Comment:
Should we use quotes for 'verbose_logging' for consistency? Not sure, just asking.

Reply:
Yes, we should. Good catch. Addressed in 98bef4add67aac7d6bc04eb223b41fffffb22b9d